zbus_names/src/interface_name.rs

```diff
@@ -184,6 +184,12 @@ impl<'name> TryFrom<Cow<'name, str>> for InterfaceName<'name> {
     }
 }
 
+impl<'name> From<InterfaceName<'name>> for Str<'name> {
```

Comment:
* It'd be strange to only implement this for InterfaceName. Please add for other `Str`-based types too, for consistency.
* Completely separate change from this commit. You know what to do. :)